### PR TITLE
Add generate_bbox

### PR DESCRIPTION
**Pull request by yuyu2172**
This function is repeatedly used in the tests.

### CODE REVIEW

File: chainercv/utils/testing/generate_bbox.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+
+def generate_bbox(n, img_size, min_length, max_length):
```
Comment (Hakuyume) [inline chainercv/utils/testing/generate_bbox.py]: How about `generate_random_bbox`?

**Comment by Hakuyume** [comment]
Perhaps, you can also use this function in the following tests.

- test_detection_vis_report.py
- test_detection_voc_evaluator.py
- test_non_maximum_suppression.py
- test_vis_bbox.py

File: chainercv/utils/testing/generate_bbox.py (hunk 1 above)
Comment (yuyu2172) [inline chainercv/utils/testing/generate_bbox.py]: That sounds better. Thanks.

File: chainercv/utils/testing/generate_random_bbox.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            of the image on which bounding boxes locate.
+        min_length (int): The minimum length of edges of bounding boxes.
+        max_length (int): The maximum length of edges of bounding boxes.
+
+    Return:
+        numpy.ndarray:
+        Coordinates of bounding boxes. Its shape is :math:`(R, 4)`. \
+        Here, :math:`R` equals :obj:`n`.
+        The second axis contains :obj:`x_min, y_min, x_max, y_max`,
+        where :obj:`x_min + min_length <= x_max < x_min + max_length` and
```
Comment (Hakuyume) [inline chainercv/utils/testing/generate_random_bbox.py]: I think math is better than obj.

File: chainercv/utils/testing/generate_random_bbox.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+def generate_random_bbox(n, img_size, min_length, max_length):
+    """Generate valid bounding boxes with random position and shape.
+
+    Args:
+        n (int): The number of bounding boxes.
+        img_size (tuple): A tuple of length 2. The width and the height
+            of the image on which bounding boxes locate.
+        min_length (int): The minimum length of edges of bounding boxes.
```
Comment (Hakuyume) [inline chainercv/utils/testing/generate_random_bbox.py]: I guess this param can be float.

File: tests/links_tests/model_tests/faster_rcnn_tests/test_faster_rcnn_train_chain.py [hunk 4]
```diff
@@ -41,17 +33,18 @@ def setUp(self):
...
     def check_call(self):
         xp = self.link.xp
 
-        n_bbox = 3
-        imgs = chainer.Variable(_random_array(xp, (1, 3, 600, 800)))
-        bboxes = chainer.Variable(
-            _generate_bbox(xp, n_bbox, (600, 800), 16, 350)[np.newaxis])
-        labels = xp.random.randint(0, self.n_fg_class, size=(n_bbox,))
-        labels = chainer.Variable(labels[np.newaxis].astype(np.int32))
         scale = chainer.Variable(xp.array(1.))
```
Comment (Hakuyume) [inline tests/links_tests/model_tests/faster_rcnn_tests/test_faster_rcnn_train_chain.py]: Why did you leave `scale` alone?

File: chainercv/utils/testing/generate_random_bbox.py [hunk 5]
```diff
@@ -8,16 +8,18 @@ def generate_random_bbox(n, img_size, min_length, max_length):
...
     Return:
         numpy.ndarray:
         Coordinates of bounding boxes. Its shape is :math:`(R, 4)`. \
         Here, :math:`R` equals :obj:`n`.
-        The second axis contains :obj:`x_min, y_min, x_max, y_max`,
-        where :obj:`x_min + min_length <= x_max < x_min + max_length` and
-        :obj:`y_min + min_length <= y_max < y_min + max_length`.
+        The second axis contains :math:`x_{min}, y_{min}, x_{max}, y_{max}`,
+        where
+        :math:`x_{min} + min\_length \\leq x_{max} < x_{min} + max\_length`
```
Comment (Hakuyume) [inline chainercv/utils/testing/generate_random_bbox.py]: I think `min_length <= x_max - xmin < max_length` is better.

Comment (yuyu2172) [inline chainercv/utils/testing/generate_random_bbox.py]: OK

**Comment by Hakuyume** [comment]
LGTM

**Pull request merged**